packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -496,6 +498,36 @@ class ServiceConnectionManager {
   }
 }
 
+class ResolvedUriManager {
+  final _resolvedUrlMap = <String, String?>{};
+  final _unknownUris = <String>[];
```

Comment:
instead of tracking unknown uris here, track these elsewhere at the call site of `fetchUnknownUris`. This manager knows that it does not know any uri that is not in `_resolvedUrlMap`, so no need to track the list here.